test/integration/controller/mpijob/mpijob_controller_test.go

```diff
@@ -203,6 +203,12 @@ var _ = ginkgo.Describe("Job controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 			return len(createdWorkload.Status.Conditions) == 1
 		}, util.ConsistentDuration, util.Interval).Should(gomega.BeTrue())
 
+		// We need to set startTime to the job since the mpijob-controller doesn't exist in envtest.
+		ginkgo.By("setting startTime to the job")
+		now := metav1.Now()
+		createdJob.Status.StartTime = &now
+		gomega.Expect(k8sClient.Status().Update(ctx, createdJob)).Should(gomega.Succeed())
+
```

Comment:
With #868 I propose to only do the update of the status for batch/job since only there it has an actual impact. Maybe we should not check this.

Reply:
+1

Please rebase

Reply:
done.